specification/resources/resource-manager/Microsoft.Resources/stable/2016-09-01/links.json

```diff
@@ -386,6 +386,14 @@
       "required": true,
       "type": "string",
       "description": "The API version to use for the operation."
+    },
+    "ApiVersionParameterExplicit": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "The API version to use for the operation.",
+      "x-ms-api-version": false
```

Comment:
What's the purpose to add a new api version parameter? What's this extension used for? I didn't find this from [autorest extension document](https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md).

Reply:
this is to enable generating explicit api_version parameter autorest.python v5.
in v3 parameter was always generated, in v5 it's not unless i add x-ms-api-version: false
this has no effect on autorest.python v3